### DISCUSSION TITLE
how do I disable rollup's visualiser plugin when building?

### DISCUSSION THREAD

**Discussion by orzechowskid** [storybookjs/storybook#25680] [Help] [answered]
### Summary

running `npx storybook build` in my react+vite app results in a web browser window opening up which contains a visualization of bundle chunks and sizes.  how do I disable `rollup-plugin-visualizer` to prevent this browser window from appearing?  the build command's `--quiet` flag did not seem to have an effect, nor did setting the `CI` environment variable to `true`.

### Additional information

```
// .storybook/main.ts

import { type StorybookConfig } from "@storybook/react-vite";

const config: StorybookConfig = {
  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@storybook/addon-onboarding',
    '@storybook/addon-interactions',
  ],
  framework: {
    name: '@storybook/react-vite',
    options: {},
  },
  docs: {
    autodocs: 'tag',
  },
}

export default config;
```

**Comment by mehmethalis** [comment, marked as answer]
Hi Dan,
You can use [viteFinal](https://storybook.js.org/docs/api/main-config-vite-final) option in storybook config. When i want to exclude a plugin, i use that.

``` import { withoutVitePlugins } from '@storybook/builder-vite'``` 
```typescript
async viteFinal(config) {
    config.plugins = await withoutVitePlugins(config.plugins, ['vite:dts'])
    return mergeConfig(config, {
      base: '',
    })
  },
```
But you need to learn plugin name in config object, you can look the plugins, when you write to console :)
